sales_disable_inventory_check/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 Komit <http://komit-consulting.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sales Disable Inventory Check",
+    "summary": "Disable warning 'Not enough inventory' when there isn't enough"
+               " product stock",
+    "version": "11.0.1.0.0",
+    "category": "Sale",
+    "website": "https://github.com/OCA/sale-workflow",
+    "author": "Phuc.nt, "
```

Comment:
For author, can you put your name and company on the same line "Nguyen Tan Phuc (komit-consulting.com)"